dask/array/core.py

```diff
@@ -3727,13 +3774,86 @@ def _check_regular_chunks(chunkset):
     return True
 
 
-def from_delayed(value, shape, dtype=None, meta=None, name=None):
-    """Create a dask array from a dask delayed value
+def from_delayed(
+    value,
+    *,
```

Comment:
Note a minorly breaking API change (you can't pass `dtype`, `meta`, etc positionally anymore). I could certainly avoid it, but I feel like these should be kwarg-only anyway (to avoid exactly this problem in the future). Not sure what our policy is for these sorts of things.

cc @jrbourbeau

Reply:
Probably best to catch any positional args and do a deprecation cycle.